Source/WebKit/NetworkProcess/NetworkSession.h

```diff
@@ -420,6 +420,9 @@ class NetworkSession : public WebCore::SWServerDelegate, public CanMakeCheckedPt
     String m_webContentRestrictionsConfigurationFile;
 #endif
     Markable<WTF::UUID> m_dataStoreIdentifier;
+
+private:
+    WebSWOriginStore* swOriginStore() const;
```

Comment:
Why not `protected` with everything else? I don't see a reason for this to be private when the data members are `protected`.